roles/openshift_facts/library/openshift_facts.py

```diff
@@ -730,6 +730,9 @@ def set_manageiq_facts_if_unset(facts):
             raise OpenShiftFactsInternalError(
                 "Invalid invocation: The required facts are not set"
             )
+    if 'use_manageiq' in facts['common']:
+        if facts['common']['use_manageiq'] is not None:
+            return facts
```

Comment:
I think I would rather see something like this instead of returning early:
```
if 'use_mangeiq' not in facts['common']:
    facts['common']['use_manageiq'] = facts['common']['version_greater_than_3_1_or_1_1']
```

Since we are not setting a default for `use_manageiq` before `set_manageiq_facts_if_unset()` then we would only set the default if it is not currently set.

@abutcher added some groundwork for validation with `validate_local_facts()`, it probably wouldn't hurt to test that  facts['common']['use_manageiq'] is indeed a boolean like we expect there instead of testing against None in this method.